inference-engine/src/transformations/src/transformations/common_optimizations/relu_fake_quantize_fusion.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/relu_fake_quantize_fusion.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ReluFakeQuantizeFusion, "ReluFakeQuantizeFusion", 0);
+
+ngraph::pass::ReluFakeQuantizeFusion::ReluFakeQuantizeFusion() {
+    auto data_pattern = ngraph::pattern::any_input();
```

Comment:
Please add CC macros. You can find examples in the PR https://github.com/openvinotoolkit/openvino/pull/3795